### DISCUSSION TITLE
Websockets (Socket.io) and Adonis v6

### DISCUSSION THREAD

**Discussion by AmreshSinha** [adonisjs/core#4407] [Help] [answered]
Hey 👋 

I recently started with Adonis v6.
Although I can see guides and cookbooks for integrating Socket.io with version 5 and older Adonis versions, I don't seem to find any for version 6.

It's my first time using Adonis, and I wanted to integrate Websocket for my project. Although I am currently figuring it out, it would be very helpful and a time saver if anyone has incorporated it in v6 already and can share it below.

Thanks.

**Comment by imanfattahi** [comment]
Hi @AmreshSinha 

Could you find any way to do it?

**Reply by AmreshSinha** [reply, marked as answer]
https://github.com/adonisjs/core/discussions/4406

**Reply by imanfattahi** [reply]
I appreciate it.
